Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageReactionsBubbleView.swift

```diff
@@ -35,6 +35,11 @@ open class _ChatMessageReactionsBubbleView<ExtraData: ExtraDataTypes>: _View, Th
     // MARK: - Life Cycle
 
     override open func updateContent() {
+        guard content?.channel.config.reactionsEnabled == true else {
+            isHidden = true
+            return
+        }
+
```

Comment:
I believe a better place to handle this is inside `_ChatMessageLayoutOptionsResolver` and just not return the `reactions` layout option.

Reply:
@VojtaStavik @evsaev It does not seem to work. Even with hardcoding the options to always not include the reactions. The reactions are always rendered. Let's discuss on slack